src/tpt_reports/_version.py

```diff
@@ -1,2 +1,2 @@
 """This file defines the version of this module."""
-__version__ = "1.0.0"
+__version__ = "1.1.0-rc.1"
```

Comment:
Sorry I'm a little late to the party here:

This is a big version jump for this extremely minor change.  Also, the version was never finalized prior to merging (i.e. it should have been finalized to `1.1.0`).  

I'd have expected this small PR to be only be a `patch` update putting the version to `1.0.1`.  Not a big deal now since it's already been merged, but the next PR for this repo should definitely include a finalized release version before being merged (this is precisely why we have the "Finalize version" checkbox in the PR description).

Reply:
I see that @ameliav finalized the version in https://github.com/cisagov/tpt-reports/pull/68, so good job there. 👍